extension/src/client/GradleClient.ts

```diff
@@ -223,6 +226,37 @@ export class GradleClient implements vscode.Disposable {
     );
   }
 
+  public async getDependencies(projectDir: string, gradleConfig: GradleConfig): Promise<DependencyItem | undefined> {
+    const request = new GetDependenciesRequest();
+    request.setProjectDir(projectDir);
+    request.setGradleConfig(gradleConfig);
+    const getDependenciesStream = this.grpcClient!.getDependencies(request);
+    try {
+      return await new Promise((resolve, reject) => {
+        let reply: GetDependenciesReply | undefined;
+        getDependenciesStream
+          .on('data', async (getDependenciesReply: GetDependenciesReply) => {
+            reply = getDependenciesReply;
+          })
+          .on('error', reject)
+          .on('end', () => { resolve(reply?.getItem()); });
+      });
+    } catch (err) {
+      logger.error(
+        `Error getting dependencies for ${projectDir}: ${
+          err.details || err.message
+        }`
+      );
+      this.statusBarItem.command = COMMAND_SHOW_LOGS;
+      this.statusBarItem.text = '$(warning) Gradle: Build Error';
+      this.statusBarItem.show();
+    } finally {
+      process.nextTick(() =>
```

Comment:
what's the `nextTick` trick for?

Reply:
Here I just follow the logics from `getBuild`, and I'll consider about whether we really need it again.